examples/data_sharing.tex

```diff
@@ -0,0 +1,210 @@
+\documentclass[a4paper]{article}
+\usepackage{data}
+
+%%%%% Now we input our variable terms as glossary entries %%%%%
+
+\newglossaryentry{type}{
+	name={Company},
+	first={the ``Company''},
+	text={Company},
+	description={is a commercial undertaking}
+}
+
+\newglossaryentry{undertaking}{
+	name={King of the Eagles},
+	first={King of the Eagles, hereafter `\glsentrytext{type}'},
+	text={the \glsentrytext{type}},
+	description={are Big birds. Live on a hill}
+}
+
+\newglossaryentry{processor}{
+	name={Mirkwood Elves},
+	first={Mirkwood Elves, hereafter `the Processor'},
+	text={Data Processor},
+	description={are the Elves of the Noldor Line}
+}
+
+\newcommand{\services}{Orc-slaying}
+
+\newcommand{\companyAddress}{123 Eirie Road}
+
+
+\newcommand{\processorAddress}{Third Tree, Mirkwood}
+
+\newcommand{\processorEmail}{elves@place.mw}
+
+\title{Data Sharing Agreement between \Glsentrytext{undertaking} \& \Glsentrytext{processor}}
+
+\begin{document}
+
+\begin{titlepage}
+
+\centering
+
+\vspace{0.4\textheight}
+
+\vspace{3cm}
+
+\scshape{\Huge\Glsfmtname{undertaking}}
+
+\vspace{0.1\textheight}
+
+\Large\scshape{and}
+
+\vspace{0.1\textheight}
+
+\scshape{\Huge\Glsfmtname{processor}}
```

Comment:
> @passamo9: The Party names on the title page should be the same font size as the rest of the text.

Reply:
This is fixed in 3a40e212afe7085d039a on my branch.

![image](https://user-images.githubusercontent.com/52453904/108002428-e254b080-6fef-11eb-96c0-e90911feb82b.png)